### PR TITLE
List view

### PR DESCRIPTION
**Pull request by tysnj**
## Is this a fix or a feature?
Both

## What is the change?
Add responsive design. Style main page. Add hover effects to individual cards.

## What does it fix?
Lack of responsive design.

## Where should the reviewer start?
`list-view` branch.

## How should this be tested?
run `npm start` and test all functionality.

### CODE REVIEW

File: src/Components/List/List.css [hunk 1]
```diff
@@ -1,9 +1,84 @@
...
+  }
+}
+
+@media all and (min-width: 750px) {
+  .movies-container {
+    padding: 20px 50px 40px;
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    grid-gap: 1px 20px;
+    justify-content: center;
```
Comment (daomeow) [inline src/Components/List/List.css:31]: You don't need to repeat this if it is in the prior media query :)

File: src/Components/List/List.css [hunk 2]
```diff
@@ -1,9 +1,84 @@
...
+    align-content: center;
```
Comment (daomeow) [inline src/Components/List/List.css:32]: Same here!

**Comment by daomeow** [comment]
WOW, way to take it to the next level with tooltip!

File: src/Components/List/List.css (hunk 1 above)
Comment (tysnj) [reply src/Components/List/List.css]: I was able to remove some of these repetitive styles. Thanks for the tip!

File: src/Components/List/List.css (hunk 2 above)
Comment (tysnj) [reply src/Components/List/List.css]: Addressed!

**Pull request merged**